src/ObjCRuntime/Runtime.cs

```diff
@@ -1356,4 +1356,48 @@ public int GetHashCode (Type obj)
 			return obj.GetHashCode ();
 		}
 	}
+
+	internal struct IntPtrTypeValueTuple : IEquatable<IntPtrTypeValueTuple>
+	{
+		private static readonly IEqualityComparer<IntPtr> item1Comparer = Runtime.IntPtrEqualityComparer;
+		private static readonly IEqualityComparer<Type> item2Comparer = Runtime.TypeEqualityComparer;
+
+		public readonly IntPtr Item1;
+		public readonly Type Item2;
+
+		public IntPtrTypeValueTuple(IntPtr item1, Type item2)
```

Comment:
Style: there should be a space before the opening parenthesis (this occurs several times in this class).